app/templates/components/forms/session-speaker-form.hbs

```diff
@@ -239,12 +243,16 @@
           {{/if}}
           {{#if (eq field.type 'image')}}
             {{widgets/forms/image-upload
-              imageUrl=(mut (get data.speaker field.fieldIdentifier))
-              label=(t 'Logo')
-              id=(if field.isRequired (concat 'speaker_' field.fieldIdentifier '_required') (concat 'speaker_' field.fieldIdentifier))
-              icon='image'
-              hint=(t 'Select an image')
-              maxSizeInKb=1000}}
+                imageUrl=(mut (get data.speaker field.fieldIdentifier))
+                needsCropper=true
```

Comment:
why?

Reply:
@iamareebjamal  cropper implemantation in wizard/basic-details.hbs is like following  -
```
{{widgets/forms/image-upload
        imageUrl=data.event.originalImageUrl
        needsCropper=true
        label=(t 'Event Image')
        id='event_image'
        icon='camera'   
        hint=(t 'Select Event Image')
        maxSizeInKb=10000
        helpText=(t 'We recommend using at least a 2160x1080px (2:1 ratio) image')
        requiresDivider=true}}
```

Reply:
You have added needsCropper=true

See the line of my review and try to understand my question

It's obsolete

Reply:
So, I should remove '`needCropper` ?

Reply:
@kushthedude ,if i remove needsCropper it is not working?